package.json

```diff
@@ -26,5 +26,8 @@
     "eslint": "^8.15.0",
     "eslint-config-google": "^0.14.0",
     "jest": "^28.1.0"
+  },
+  "dependencies": {
+    "remedian": "^0.2.0"
```

Comment:
*nitpick:* please, remove this dependency. It is unnecessary and kinda circular. Also, `package-lock.json` should not have any changes.